MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -936,22 +937,32 @@ private void BuildStreamVideoItem(
             }
 
             // Prefer matching audio codecs, could do better here
-            var audioCodecs = ContainerProfile.SplitValue(audioCodec);
+            List<string> audioCodecs = [];
+            if (audioCodec is not null)
+            {
+                audioCodecs.AddRange(ContainerHelper.Split(audioCodec));
+            }
```

Comment:
Btw, is this work? But we probably lose the benefit of the `List'.
```suggestion
            List<string> audioCodecs = ContainerHelper.Split(audioCodec);
```